mqr/utils.py

```diff
@@ -70,3 +71,49 @@ def get_next_send_date():
     # TODO: maybe this should be smarter
     # calculate date based on edd_or_dob and next week, incase message failed?
     return get_today() + timedelta(weeks=1)
+
+
+def get_facility_province(facility_code):
+    try:
+        clinic_code = ClinicCode.objects.get(code=facility_code)
+    except ClinicCode.DoesNotExist:
+        clinic_code = None
+
+    return clinic_code
+
+
+def get_weeks_pregnant(edd):
+    full_term_weeks = 40
+    estimated_date = edd
+
+    if isinstance(edd, str):
```

Comment:
I think if we make the serializer field a date we wont need this

Reply:
Done